tests/storage.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { test } from "@playwright/test";
+import {
+  createPool,
+  deletePool,
+  editPool,
+  randomPoolName,
+  savePool,
+} from "./helpers/storagePool";
+import {
+  createVolume,
+  deleteVolume,
+  editVolume,
+  randomVolumeName,
+  saveVolume,
+} from "./helpers/storageVolume";
+
+test("storage pool create, edit and remove", async ({ page }) => {
+  const pool = randomPoolName();
+  await createPool(page, pool);
+
+  await editPool(page, pool);
+  await page.getByPlaceholder("Enter description").fill("A-new-description");
+  await savePool(page);
+
+  await page.getByTestId("tab-link-Overview").click();
+  await page.getByText("DescriptionA-new-description").click();
+
+  await deletePool(page, pool);
+});
+
+test("storage volume create, edit and remove", async ({ page }) => {
+  const volume = randomVolumeName();
+  await createVolume(page, volume);
+
+  await editVolume(page, volume);
+  await page.getByPlaceholder("Enter value").fill("50");
```

Comment:
Setting a 50 GiB size just for a test seems a bit risky. Maybe I'm paranoid, but I would set a smaller value, just in case. 5 is probably enough.

Reply:
The default is depending on the total size (for me, it is 30GB) and 50 ensures we do only extend it, because shrinking is not allowed.
I changed the creation to say 1GB, and here we put 2GB.